src/platform/iotjs_systemio-linux.c

```diff
@@ -36,106 +36,108 @@ bool DeviceCheckPath(const char* path) {
     return false;
   }
 
-  DDDLOG("DeviceCheckPath() - path exist");
+  DDDLOG("%s - path exist", __FUNCTION__);
 
   return true;
 }
 
 
-bool DeviceOpenWriteClose(const char* path, char* value) {
+bool iotjs_systemio_open_write_close(const char* path, char* value) {
```

Comment:
What about `const char* value`, instead of `char*`. The variable value is used for read only.